cmd/rewards-api/main.go

```diff
@@ -305,8 +310,12 @@ func main() {
 		defer definitionsConn.Close()
 
 		definitionsClient := pb_defs.NewDeviceDefinitionServiceClient(definitionsConn)
+		chClient, err := ch.NewClient(&settings)
+		if err != nil {
+			logger.Fatal().Err(err).Msg("Failed to create ClickHouse client.")
+		}
```

Comment:
Can we use the same client we already created above?

Reply:
I put it in both spots because of the case switch-- we only use it in the api and the rewards calculation job but I agree that I don't like having it defined twice in the same file